drivers/mipi_dbi/mipi_dbi_stm32_fmc.c

```diff
@@ -0,0 +1,203 @@
+/*
+ * Copyright (c) 2024 Bootlin
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#define DT_DRV_COMPAT st_stm32_fmc_mipi_dbi
+
+#include <zephyr/drivers/gpio.h>
+#include <zephyr/drivers/mipi_dbi.h>
+#include <zephyr/drivers/clock_control/stm32_clock_control.h>
+#include <zephyr/sys/barrier.h>
+#include <zephyr/sys/sys_io.h>
+#include <zephyr/sys/byteorder.h>
+
+#include <zephyr/logging/log.h>
+LOG_MODULE_REGISTER(mipi_dbi_stm32_fmc, CONFIG_MIPI_DBI_LOG_LEVEL);
+
+struct mipi_dbi_stm32_fmc_config {
+	/* Reset GPIO */
+	const struct gpio_dt_spec reset;
+	/* Power GPIO */
+	const struct gpio_dt_spec power;
+	mem_addr_t register_addr;
+	mem_addr_t data_addr;
+	uint32_t fmc_AddressSetupTime;
+	uint32_t fmc_DataSetupTime;
+	uint32_t fmc_MemoryWidth;
```

Comment:
Coding guidelines, please use snake case.
I won't block, but this would be nice to fix.